examples/configs/evaluation/hf_evaluate_config.yaml

```diff
@@ -0,0 +1,65 @@
+name: "lm-buddy-hf-evaluate"
+
+# Input dataset path
+dataset:
+  path: "s3://platform-storage/datasets/dialogsum"
+
+
+# Settings specific to the hf_evaluate entrypoint
+evaluation:
+  # metrics to be used for the evaluation
+  # (you can add "rouge", "meteor", and "bertscore" atm)
+  metrics: ["rouge", "meteor"]
+  # enable/disable tqdm to track eval progress
+  # (useful when running interactively, noisy on ray logs)
+  enable_tqdm: True
+  # rely on HF pipeline for summarization (ignored if using OAI API)
+  use_pipeline: True
+  # perform inference / evaluation on the first max_samples only
+  max_samples: 10
+  # output file path
+  # - if you provide a path complete with a filename, results will be stored in it
+  # - if you provide a dir, results will be stored in <dir>/<config.name>/eval_results.json
+  # - if you don't provide a storage path, results will be stored locally (see ~/.lm-buddy/results)
+  # storage_path: "s3://platform-storage/experiments/results/"
+
+# Model to evaluate. Choose one of the following options by uncommenting it
```

Comment:
You mean switch between differente models? I mainly wanted to provide a configuration showcasing all the possibilities, because we have been supporting different models (in other entrypoints too) but a single example is not very self-explicative.
I think cli params are a potential alternative, wdyt would work best if we think about running these from mzai-platform?

Reply:
Will address this in one of the follow-up comments!